docs/source/getting-started/assessing-utility/composition.ipynb

```diff
@@ -0,0 +1,742 @@
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (1, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>DP Mean Hours Worked</th><th>DP Median Hours Worked</th></tr><tr><td>f64</td><td>i64</td></tr></thead><tbody><tr><td>41.350777</td><td>54</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (1, 2)\n",
+       "┌──────────────────────┬────────────────────────┐\n",
+       "│ DP Mean Hours Worked ┆ DP Median Hours Worked │\n",
+       "│ ---                  ┆ ---                    │\n",
+       "│ f64                  ┆ i64                    │\n",
+       "╞══════════════════════╪════════════════════════╡\n",
+       "│ 41.350777            ┆ 54                     │\n",
+       "└──────────────────────┴────────────────────────┘"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dp_hours = context.query().select(\n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.mean((1,45)).alias(\"DP Mean Hours Worked\"), \n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.median(candidates).alias(\"DP Median Hours Worked\"),\n",
+    ").release().collect()\n",
+    "\n",
+    "dp_hours"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interactive Sequential Composition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In cases where you want to apply interactive sequential composition, you can use the `dp.c.make_sequential_composition` method. There are two ways to implement sequential composition: using transformations and measurements directly and passing in polars queries. \n",
+    "\n",
+    "Also note, that the following queries are interactive, but they should not be assumed to be adaptive. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Interactive Sequential Composition with Transformations and Measurements"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The interactive sequential composition doesn't use the compositor we defined at the start of this notebook. Instead, we construct a composition through transformations and measurements, combinator elements present in lower-level frameworks in the OpenDP library. \n",
+    "\n",
+    "We will also define a `queryable` in the following code. It can essentially be thought of as a class that executes the queries and tracks the privacy expenditure. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Privacy Consumption of Entire Composition: 4.0\n"
```

Comment:
Enclose these kinds of outputs in a box or something again for more clarity and separation from the main text.